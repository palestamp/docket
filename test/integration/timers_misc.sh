rm ~/.timer
set -v
docket timer new work --abstract
docket timer new home
docket timer new work_child
docket timer new work_child2 --parent work
docket timer set work_child parent work
docket timer set work_child event:onstart:stop work_child2

docket timer new home_child --parent home
docket timer set home_child event:onstop:start work_child

docket timer start work_child2
docket timer stat --status-only
docket timer start home_child
docket timer stat --status-only
docket timer stop home_child
docket timer stat --status-only
