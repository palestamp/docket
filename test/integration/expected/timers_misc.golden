docket timer new work --abstract
docket timer new home
docket timer new work_child
docket timer new work_child2 --parent work
docket timer set work_child parent work
docket timer set work_child event:onstart:stop work_child2

docket timer new home_child --parent home
docket timer set home_child event:onstop:start work_child

docket timer start work_child2
docket timer stat --status-only
*               running   
work            running   
home            stopped   
work_child      stopped   
work_child2     running   
home_child      stopped   
docket timer start home_child
docket timer stat --status-only
*               running   
work            running   
home            running   
work_child      stopped   
work_child2     running   
home_child      running   
docket timer stop home_child
docket timer stat --status-only
*               running   
work            running   
home            running   
work_child      running   
work_child2     stopped   
home_child      stopped   
